client/Cargo.toml

```diff
@@ -37,6 +37,7 @@ serde = { version = "1.0.151", features = ["derive"] }
 serde_json = "1.0.91"
 base64 = "0.13.1"
 thiserror = "1.0.38"
+displaydoc = "0.2.4"
```

Comment:
What do you think about making it a workspace depenency? We already have an issue for that, so I think we can start adding new depenencies as workspace's.

That way you won't have to specify all options like version everytime

Reply:
I filed #3708 for that yesterday =)
Probably makes sense to make it here too (there will be merge conflicts anyway..)